media/js/src/assetDetail/ViewItem.jsx

```diff
@@ -34,34 +34,85 @@ export default class ViewItem extends React.Component {
             userIsAuthor = authorId === window.MediaThread.current_user;
         }
 
+        const references = [];
         if (this.state.references) {
             this.state.references.forEach(function(reference, idx) {
                 references.push(
-                    <div className="card w-100 card-selection" key={idx}>
-                        <div className="card-body">
-                            <h5 className="card-title">
-                                <div>
-                                    {reference.title}
-                                </div>
-                            </h5>
-                            <h6>
-                                Assignment
-                            </h6>
-                            <p className="card-text">
-                                Collected 12/19/15 01:02 AM
-                            </p>
-                            <a href="#" className="btn btn-primary btn-sm">
-                                View
-                            </a>
-                        </div>
-                    </div>
+                    <li key={idx}>
+                        <a href={reference.url}>{reference.title}</a>
+                    </li>
                 );
             });
         }
 
+        let tags = '(None)';
+        if (
+            this.props.asset &&
+                this.props.asset.global_annotation &&
```

Comment:
I'm not sure we want to pull the tags & terms off the global_annotation, as that idea is being discontinued. Can we show all the tags & terms from the visible selections on an item?

Reply:
Ah okay - yeah I can definitely pull tags off the item's selections. I did realize global_annotation is getting deprecated, I just didn't see anywhere else tags were present on the item.

Reply:
Tags & terms aren't present on the item itself. The `global_annotation` is a weird "special" annotation that's created to signify the item is within the user's collection. Tags, terms and notes for the "whole item" were kept on the `global_annotation`. Since we've deprecated the idea of adding tags, terms and notes to the item, we can kill the `global_annotation`, and rewrite the collection api to show any items a user owns, e.g. uploaded or imported, and any items a user has annotated. Future work to do.